sdk/resourcemanager/azure-resourcemanager-compute/src/main/java/com/azure/resourcemanager/compute/models/VirtualMachine.java

```diff
@@ -2400,6 +2456,55 @@ interface WithOSDisk {
              */
             Update withOSDisk(Disk disk);
         }
+
+        /** The stage of the VM update allowing to change security features. */
+        interface WithSecurityFeatures {
+            /**
+             * Enables secure boot feature.
+             * <p>
+             * Note:
+             * Your VM's security type should be set in order to enable this feature.
+             * After changing security features, a restart is required for your VM to take effect.
+             * </p>
+             *
+             * @return the next stage of the update
+             */
+            Update withSecureBoot();
+
+            /**
+             * Disables secure boot feature.
+             * <p>
+             * Note:
+             * After changing security features, a restart is required for your VM to take effect.
+             * </p>
+             *
+             * @return the next stage of the update
+             */
+            Update withoutSecureBoot();
+
+            /**
+             * Enables vTPM feature.
+             * <p>
+             * Note:
```

Comment:
nit, since it already in Javadoc, "Note:" might not be necessary.

Reply:
Removed.